### PR TITLE
Fix so that dump logical prints everything without a regex

### PR DESCRIPTION
**Pull request by martinfurmanski**

### CODE REVIEW

File: tools/src/main/java/org/neo4j/tools/dump/DumpLogicalLog.java [hunk 1]
```diff
@@ -128,15 +128,24 @@ public LogVersionedStoreChannel next( LogVersionedStoreChannel channel ) throws
...
+            while ( entryCursor.next() )
             {
-                for ( LogEntry entry : cursor.get() )
+                out.println( entryCursor.get().toString( timeZone ) );
+            }
+        }
+        else
+        {
+            TransactionLogEntryCursor transactionCursor = new TransactionLogEntryCursor( entryCursor );
+            try ( IOCursor<LogEntry[]> cursor = new FilteringIOCursor<>( transactionCursor, new TransactionRegexCriteria( regex, timeZone ) ) )
```
Comment (burqen) [inline tools/src/main/java/org/neo4j/tools/dump/DumpLogicalLog.java]: Format, line too long. And while on it, line 127 could use a line break as well.

**Review by burqen — CHANGES_REQUESTED** [review]
Looks good! Small format comment.

**Comment by burqen** [comment]
@martinfurmanski Friendly ping about review being finished.

**Comment by martinfurmanski** [comment]
@burqen Friendly ping about PR ready to merge.

**Review by burqen — APPROVED** [review]

**Pull request merged**